fe2-android/app/src/test/unit/java/com/github/dedis/popstellar/utility/handler/ElectionHandlerTest.java

```diff
@@ -200,11 +208,20 @@ private void handleElectionResults(Set<QuestionResult> results, Channel election
     messageHandler.handleMessage(messageSender, electionChannel, message);
   }
 
+  private void setMessageRepoReturns(MessageGeneral message) {
+    messageRepo = mock(MessageRepository.class);
```

Comment:
@GabrielFleischer do you think it would be better to use Hilt mock injection ?

Reply:
It's true that this is not perfect.

I don't think you need a mock here, you could simply call 
`messageRepo.addMessage(message);`

As the message repository is a simple class, mocking it comes down to reimplementing it.

On line 111, you already added an instantiation of the base class, you should simply use it 😄